src/main/java/uk/gov/hmcts/et/common/model/bulk/types/DynamicFixedListType.java

```diff
@@ -6,49 +6,102 @@
 import org.apache.commons.collections4.CollectionUtils;
 import org.apache.commons.lang3.StringUtils;
 
+import java.util.ArrayList;
 import java.util.List;
 import java.util.Objects;
 import java.util.Optional;
+import java.util.stream.Collectors;
 
 @JsonIgnoreProperties(ignoreUnknown = true)
 @Data
 public class DynamicFixedListType {
 
-    private transient DynamicValueType value;
+    private DynamicValueType value;
     @JsonProperty("list_items")
     private List<DynamicValueType> listItems;
 
     public static DynamicFixedListType of(DynamicValueType value) {
         Objects.requireNonNull(value);
-        var dynamicFixedListType = new DynamicFixedListType();
+        DynamicFixedListType dynamicFixedListType = new DynamicFixedListType();
         dynamicFixedListType.value = value;
         return dynamicFixedListType;
     }
 
     public static DynamicFixedListType from(List<DynamicValueType> listItems) {
-        var dynamicFixedListType = new DynamicFixedListType();
+        DynamicFixedListType dynamicFixedListType = new DynamicFixedListType();
         dynamicFixedListType.listItems = listItems;
         return dynamicFixedListType;
     }
 
-    public static DynamicFixedListType from(List<DynamicValueType> listItems, DynamicFixedListType original) {
-        var dynamicFixedListType = DynamicFixedListType.from(listItems);
+    public static DynamicFixedListType from(List<DynamicValueType> selectListItems, DynamicFixedListType original) {
```

Comment:
If we're making this level of changes, let's get JavaDoc in here :)
It's probably worth also adding it at class level too.

Reply:
Agreed about adding JavaDoc. Also, I think it is more clear if I add a brief JavaDoc at class level and one for this static method separately. It will be more focused.